bokeh/models/widgets/sliders.py

```diff
@@ -202,6 +202,25 @@ def value_as_datetime(self):
             d2 = datetime.utcfromtimestamp(v2 / 1000)
         else:
             d2 = v2
+        return d1, d2    \
+
+    @property
+    def value_as_date(self):
+        ''' Convenience property to retrieve the value tuple as a tuple of
+        date objects.
+
```

Comment:
We should add a note here that states that this function was added in version 1.1

Reply:
Can this literally be 'Added in version 1.1'? I can't immediately find an example when looking through previous releases.

Reply:
That's because this is one of the first additions since 1.0 :) I think that text on a line by itself would be fine.